cmd/conform/serve.go

```diff
@@ -44,35 +44,35 @@ var serveCmd = &cobra.Command{
 			}
 
 			go func() {
-				dir, err := os.CreateTemp("/tmp", "conform")
+				dir, err := os.MkdirTemp("/tmp", "conform")
```

Comment:
I would do to make it respect TMPDIR

```suggestion
				dir, err := os.MkdirTemp("", "conform")
```